docs/enhancements/0002_day0-integration.md

```diff
@@ -0,0 +1,282 @@
+# Day-0 integration with KMM
+
+### Motivation
+
+KMM is managing mostly day2 kmods in OCP. It also offers a library for generating
+systemd files for helping customers to deploy their day-1 kmod.
+
+What KMM doesn't offer today to customers is the deployment and management of
+day-0 kmods.
+
+Day-0 kmod are complex as they need to be present in the system at boot time,
+even before OCP is fully installed and certainly before KMM is deployed to the
+cluster, therefore, there is not much that KMM can do to help deploy those kmods but
+it can manage their lifecycle once a customer has installed a cluster with his
+day-0 kmods in it - this is what this enhancement is about.
+
+### Day-0 cluster background
+
+A customer wishing to deploy OCP with custom nodes will need to generate 2
+artifacts using [image-builder](https://www.redhat.com/en/topics/linux/what-is-an-image-builder)
```

Comment:
does image builder capable already to generate a custom iso?

Reply:
No.

Reply:
ok, so how will this enhancement help? :)

Reply:
The ISO building capability will reach image-builder/osbuild at some point :)